frontend/ui/src/pages/Channels/ChannelsMainPage/SourceInfo/index.tsx

```diff
@@ -60,9 +57,9 @@ const SourceInfo = (props: SourceInfoProps) => {
                   );
                 })}
               </div>
-              <div className={styles.extraChannel} onClick={props.onSourceInfoClick}>
+              <div className={styles.extraChannel} onClick={() => props.onSourceInfoClick()}>
```

Comment:
we can use `props.onSourceInfoClick` and it will also work, with less render cycles.